src/main/lua/io/piulxio/defs.lua

```diff
@@ -0,0 +1,129 @@
+--------------------------------------
+-- IO device identifier
+--------------------------------------
+IO_PIULXIO_IDENTIFIER = "piulxio"
+
+--------------------------------------
+-- Digital input states
+--------------------------------------
+-- Panels P1 (if any sensor pressed)
+IO_LXIO_DI_P1_LU = 0
```

Comment:
I suggest to align the prefix for these fields to the ones in the piumk6 module, i.e. `IO_PIULXIO_`

Reply:
Will do